cmake/Platform/Common/MSVC/Configurations_msvc.cmake

```diff
@@ -12,18 +12,12 @@ if(NOT CMAKE_GENERATOR MATCHES "Visual Studio 1[6-7]")
     message(FATAL_ERROR "Generator ${CMAKE_GENERATOR} not supported")
 endif()
 
-# For the other cases, verify that it wasn't invoked with an unsupported architecture. defaults to x86 architecture
-if(SUPPORTED_VS_PLATFORM_NAME_OVERRIDE)
-    set(SUPPORTED_VS_PLATFORM_NAME ${SUPPORTED_VS_PLATFORM_NAME_OVERRIDE})
-else()
-    set(SUPPORTED_VS_PLATFORM_NAME x64)
-endif()
-
-if(CMAKE_VS_PLATFORM_NAME AND NOT CMAKE_VS_PLATFORM_NAME STREQUAL "${SUPPORTED_VS_PLATFORM_NAME}")
-    message(FATAL_ERROR "${CMAKE_VS_PLATFORM_NAME} architecture not supported")
+# Verify that it wasn't invoked with an unsupported target/host architecture. Currently only supports x64/x64
+if(CMAKE_VS_PLATFORM_NAME AND NOT CMAKE_VS_PLATFORM_NAME STREQUAL "x64")
+    message(FATAL_ERROR "${CMAKE_VS_PLATFORM_NAME} target architecture not supported")
```

Comment:
Nit;  If the check is so specific as to check for x64 then aybe the error message should be "${CMAKE_VS_PLATFORM_NAME} is not supported, it must be 'x64'"